### PR TITLE
Update embedded_workflows.md

### PR DESCRIPTION
**Pull request by qahmed1998**

### CODE REVIEW

**Comment by qahmed1998** [comment]
@miq-bot assign @Fryguy

**Comment by qahmed1998** [comment]
@miq-bot add-label radjabov?

**Comment by qahmed1998** [comment]
@miq-bot add_labels radjabov/yes?

**Comment by qahmed1998** [comment]
@miq-bot remove_labels radjabov/no

**Comment by kbrock** [comment]
ugh. sorry. Missread and thought this was Jason's PR

File: managing_providers/_topics/embedded_workflows.md [hunk 1]
```diff
@@ -165,6 +165,8 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
 
    The workflow code must be in the Amazon States Language (ASL) format and follow its supported specifications. For more information about Amazon States Language and its specification, see [Amazon States Language Guide](https://states-language.net/).
 
+   Amazon States Language does not support **Map** and **Parallel** states.
```
[the remark this answers is not in the document]
Comment (kbrock) [reply managing_providers/_topics/embedded_workflows.md:168]: https://github.com/ManageIQ/manageiq-documentation/pull/1809

[the remark this answers is not in the document]
Comment (qahmed1998) [reply managing_providers/_topics/embedded_workflows.md:168]: Slight wording update via this PR: https://github.com/ManageIQ/manageiq-documentation/pull/1810

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit 90bbdddd17422c51decf08766a80ed2cb12a5b8a.

```text
commit 90bbdddd17422c51decf08766a80ed2cb12a5b8a
Author: Keenan Brock <keenan@thebrocks.net>
Date:   Mon Aug 26 19:00:39 2024 -0400

    Merge pull request #1808 from qahmed1998/QA-update-asl-5692
    
    Update embedded_workflows.md
    
    (cherry picked from commit 25bab73fca3eebf3a97704f9b2f88fc32b11d90c)
```

**Pull request merged**